roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -1,15 +1,20 @@
 # Dispute Coordinator
 
-This is the central subsystem of the node-side components which participate in disputes. This subsystem wraps a database which tracks all statements observed by all validators over some window of sessions. Votes older than this session window are pruned.
+This is the central subsystem of the node-side components which participate in disputes. This
+subsystem wraps a database which tracks all statements observed by all validators over some window
+of sessions. Votes older than this session window are pruned.
 
-This subsystem will be the point which produce dispute votes, either positive or negative, based on locally-observed validation results as well as a sink for votes received by other subsystems. When importing a dispute vote from another node, this will trigger participation in the dispute.
+This subsystem will be the point which produce dispute votes, either positive or negative, based on
+locally-observed validation results as well as a sink for votes received by other subsystems. When
+importing a dispute vote from another node, this will trigger participation in the dispute.
```

Comment:
Better: When a statement import makes it clear that a dispute has been raised (there are opposing votes for a candidate), the dispute coordinator will make sure the local node will participate in the dispute.